website/src/utils/doesImageExist.ts

```diff
@@ -0,0 +1,5 @@
+export const doesImageExist = (src: string) => {
+    const img = new Image();
```

Comment:
One line description for this util will be good to have.

Reply:
removed it :)